index.html

```diff
@@ -90,9 +90,9 @@ <h1 class="primary-text-header">I’m Steven Levoyer nice to see you!</h1>
   <main>
     <!--Works-->
     <section aria-label="Section of works">
-      <div class="work">
-        <div class="card-works">
-          <img class="snapshoot" src="./img/Snapshoot-1.svg">
+      <div class="work" id="Works">
+        <div class="card-works" id="card-1">
+          <img class="snapshoot" src="./img/Desktop/Snapshoot_1.svg">
           <div class="left-block">
```

Comment:
- [ ] Dear @mdg0410, Good job you have done already, For the work card it seems your design has overflowing content. This happens for a screen size >767x to around 1200px, Pleas fix this for a better layout. 

### Your design.
![Untitled](https://user-images.githubusercontent.com/69077061/215017337-d0472e2b-88d2-4cfb-b831-932303c5a992.png)

Reply:
> Correction

*The overflow in the work section was corrected by adding the* `overflow: hidden` *attribute to the flex containing the image and content.*

|Card-example|Space-card-example|
|:---------------:|:----------------:|
|![imagen](https://user-images.githubusercontent.com/90431831/215188172-974bdf07-4d02-40c3-b362-eff6311a8ad5.png)|![imagen](https://user-images.githubusercontent.com/90431831/215188293-7d05b7bb-2e6c-4c75-94f8-0380d5d23e56.png)|